CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# [v2.3.1](https://github.com/finos/perspective/releases/tag/v2.3.1)
```

Comment:
You rebased on a reverted commit. Drop this commit please [76913ac](https://github.com/finos/perspective/pull/2279/commits/76913acc2e08e2e6fde517eed83a5dcaa0cc72e9).